app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -0,0 +1,233 @@
+        }
+        .pagebuilder-overlay {
+            max-width: 50%;
+        }
+        [data-element="content"] {
+            padding-left: 17px;
+        }
+
+        .slick-dots {
+            position: absolute;
+            bottom: 0;
+            button {
+                width: 10px;
+                height: 10px;
+                border-radius: 0;
+                border: 2px solid @color-maksym__main-black;
+            }
+        }
+    }
+}
+
+/* SECTION__CATEGORIES*/
+.section__categories {
+    margin: -15px;
+    .categories {
+        &__title {
+            .lib-visually-hidden();
+        }
+    }
+
+    [data-content-type="column-line"] {
+        .categories__item {
+            margin: 15px;
+        }
+    }
+
+    [data-appearance="collage-right"]
+        .pagebuilder-banner-wrapper
+        .pagebuilder-overlay {
+        max-width: 50%;
+        height: 100%;
+    }
+    [data-element="wrapper"] {
+        display: flex;
+        align-items: center;
+    }
+    .pagebuilder-overlay {
+        padding-left: 0;
+        p {
+            font-style: italic;
+        }
+    }
+    button {
+        padding-left: 0;
+    }
+}
+
+/* SECTION__SALE*/
+.section__sale {
+    .sale__banner {
+        .pagebuilder-banner-wrapper
+            .pagebuilder-overlay:not(.pagebuilder-poster-overlay) {
+            width: 100%;
+        }
+        [data-element="wrapper"] {
+            display: block;
+            width: 100%;
+            margin: 0%;
+
+            .pagebuilder-banner-wrapper
+                .pagebuilder-overlay:not(.pagebuilder-poster-overlay) {
+                width: 100%;
+            }
+        }
+        .pagebuilder-overlay {
+            display: block;
+            width: 100%;
+            height: 100%;
+            margin: 0;
+            &:not(.pagebuilder-poster-overlay) {
+                width: 100%;
+            }
+            .pagebuilder-collage-content {
+                width: 100%;
+                display: flex;
+                justify-content: space-between;
+            }
+        }
+        [data-element="content"] {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+        }
+        .pagebuilder-banner-wrapper .pagebuilder-banner-button {
+            margin: 0;
+        }
+    }
+}
+
+.sale {
+    &__title {
+        .lib-visually-hidden();
+    }
+}
+
+/* SECTION__BRANDS */
+.section__brands {
+    .row-full-width-inner {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+    .brands__divider {
+        width: 100%;
+    }
+    .row-full-width-inner {
+        position: relative;
+    }
+    .brands {
+        &__title {
+            position: absolute;
+            top: 5px;
+            z-index: 1;
+            background-color: white;
+            margin: 0;
+            padding: 0 5px 0 5px;
+            line-height: 30px;
+            font-size: 26px;
```

Comment:
Please use font-size mixin, check it here:
/lib/web/css/source/lib/_utilities.less : line 53

Reply:
done

Reply:
done